pipelines/common/src/main/java/com/google/fhir/analytics/view/ViewSchema.java

```diff
@@ -172,8 +174,8 @@ public static List<GenericRecord> setValueInRecord(RowList rows, ViewDefinition
             }
           }
         } else {
-          if (e.getColumnInfo().isCollection() || e.getColumnInfo().getType() instanceof String) {
-            if (e.getValues() == null || e.getValues().size() < 1) {
+          if (e.getColumnInfo().isCollection() || e.getColumnInfo().getType() != null) {
```

Comment:
Why did you remove the `instanceof` constraint?

Reply:
Okay, I took a look at this and there might be an actual bug here; not sure yet but I created issue #1502 to track this. For now, please change this line as follows:
```suggestion
          if (e.getColumnInfo().isCollection()) {
```